src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -151,60 +176,60 @@ public void periodic() {
     }
   }
 
+  private void consumeChassisSpeeds(ChassisSpeeds speeds) {
```

Comment:
i know i told you to change this earlier but i think runVelocity is better if runVelocityCmd is named as such